src/pages/workspaces/workspace/Data.js

```diff
@@ -700,6 +700,18 @@ const WorkspaceData = _.flow(
                 ])
               }, sortedEntityPairs)
             ]),
+            isFeaturePreviewEnabled('workspace-data-service') && h(DataTypeSection, {
+              title: 'WDS'
+            }, [
+              div({
+                style: {
+                  display: 'flex', flexDirection: 'column', alignItems: 'flex-start',
+                  padding: '0.5rem 1.5rem', borderBottom: `1px solid ${colors.dark(0.2)}`,
+                  backgroundColor: 'white'
+                }
+              }, ['Coming soon.']),
+              div({}, '')
+            ]),
```

Comment:
`DataTypeSection` only seems to display its children when there are > 1 children, so I added the extra `div({}, '')`. This is throwaway code which will soon be replaced by data coming from WDS, so I don't mind the weirdness here.

Reply:
Maybe [this](https://github.com/DataBiosphere/terra-ui/blob/3a84398c526a71a07eae371fe22ba84a2fe56c44/src/pages/workspaces/workspace/Data.js#L225)? Probably `children` should be assumed to already be an array, so it shouldn't need to be in additional brackets (`[children]`).

Reply:
> Probably children should be assumed to already be an array

`children` should generally be treated as "an opaque data structure".
https://reactjs.org/docs/react-api.html#reactchildren